### PR TITLE
 docker outside of docker (GSI-414)

### PR DESCRIPTION
**Pull request by KerstenBreuer**
```
- details can be found here: https://github.com/devcontainers/features/tree/main/src/docker-outside-of-docker
- removed name from devcontainer json to make it static
```

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/python-3-postgres
 // Update the VARIANT arg in docker-compose.yml to pick a Python version: 3, 3.8, 3.7, 3.6
 {
-	"name": "my-microservice",
```
Comment (Cito) [inline .devcontainer/devcontainer.json:5]: What appears in the windows title or taskbar popup notification if there is no name? Are the windows for different repos still easily distinguishable?

Comment (KerstenBreuer) [reply .devcontainer/devcontainer.json:5]: At least on vscode in windows, there is no titlebar. Is that something that is breaking your workflow on mac? I only see the name in devcontainer indicator (bottom left).

Comment (Cito) [reply .devcontainer/devcontainer.json:5]: On Windows it's shown as placeholder in the search bar that appears in the title bar, and it is also visible in the task switcher.

The point is that if you have opened multiple repos in VS Code at the same time, it should be easy to switch between them based on the name of the repo.

Unfortunately I can only check next week on MacOS again.

Comment (KerstenBreuer) [reply .devcontainer/devcontainer.json:5]: OK, have never used it. Would you value this feature more than the simplicity of distributing the devcontainer.json with all its editor configurations? If so it's perfectly fine. If no, we can still put it into the architecture channel and see if somebody is vetoing this change.

Comment (Cito) [reply .devcontainer/devcontainer.json:5]: Checked on MacOS now. Similar to Windows, the name appears in the placeholder for the search field. It also appears in the "Windows" top menu where you can switch between the different VS Code instances.

When no name is configured, it appears as empyt (there is no default). So setting a name is really helpful.

However, I think I have found a solution. What about this?
```json
"name": "${localWorkspaceFolderBasename}",
```

Works perfectly for me.

Comment (KerstenBreuer) [reply .devcontainer/devcontainer.json:5]: Great thanks. Works for me as well.

**Review by Cito — APPROVED** [review]

**Pull request merged**